### DISCUSSION TITLE
KERI implementors call 10/08/2024

### DISCUSSION THREAD

**Discussion by m00sey** [WebOfTrust/keri#100] [Meetings]
**Zoom:**
https://us06web.zoom.us/j/81679782107?pwd=cTFxbEtKQVVXSzNGTjNiUG9xVWdSdz09

**Reports:**
KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
kerisse - @henkvancann
keridoc - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

**Discussion:**
- https://github.com/WebOfTrust/keripy/issues/775 (Philipp Schlarb from esatus AG)

**Comment by m00sey** [comment]
KERIpy

Open:
- https://github.com/WebOfTrust/keripy/pull/872
- https://github.com/WebOfTrust/keripy/pull/868
- https://github.com/WebOfTrust/keripy/pull/867
- https://github.com/WebOfTrust/keripy/pull/873

Merged: 
- https://github.com/WebOfTrust/keripy/pull/864
- https://github.com/WebOfTrust/keripy/pull/869
- https://github.com/WebOfTrust/keripy/pull/870
- https://github.com/WebOfTrust/keripy/pull/871

**Comment by iFergal** [comment]
We didn't get to it in last week's call, so if there's time:

> How solicited percolation of things like OOBI URLs or schemas could work peer to peer. Using query/reply or exn messages etc.  If query/reply in KERIA, a query goes to the eid agent?

**Comment by rodolfomiranda** [comment]
## KERIA
PRs:
- https://github.com/WebOfTrust/keria/pull/300
- https://github.com/WebOfTrust/keria/pull/299

Issues:
- https://github.com/WebOfTrust/keria/issues/301
- https://github.com/WebOfTrust/keria/issues/293

**Comment by pSchlarb** [comment]
I would love to join, but zoom is saying that the meeting id is wrong.

**Comment by lenkan** [comment]
# Signify-ts

Issues:

- https://github.com/WebOfTrust/signify-ts/issues/281

PRs

- No new PRs last three weeks.
